test/test_helper.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
-require 'coveralls'
-Coveralls.wear!
+# require 'coveralls'
```

Comment:
Did you commit this by accident or is this intentional?

Reply:
This PR is in draft. I have a issue with ruby 3.1 : https://github.com/Cyberwatch/exception_notification/runs/4702998693?check_suite_focus=true

Reply:
I have disable coveralls to be able to debug the test suite.